python/pyspark/sql/connect/client.py

```diff
@@ -141,7 +141,7 @@ def _build_metrics(self, metrics: "pb2.Response.Metrics") -> typing.List[PlanMet
     def sql(self, sql_string: str) -> "DataFrame":
         return DataFrame.withPlan(SQL(sql_string), self)
 
-    def collect(self, plan: pb2.Plan) -> pandas.DataFrame:
+    def _toPandas(self, plan: pb2.Plan) -> pandas.DataFrame:
```

Comment:
Actually we should use snake naming rule for private methods. Only API follows the camel naming conversion

Reply:
oh I see done!

I am still learning how to write python code for spark....